Gems/Multiplayer/Code/Source/MultiplayerStats.cpp

```diff
@@ -6,6 +6,8 @@
  *
  */
 
+#include <AzCore/Metrics/IEventLogger.h>
+#include <AzCore/Metrics/IEventLoggerFactory.h>
```

Comment:
These includes aren't used in this file.

Reply:
Cleaned up.